src/reducers/Entities/SensorReducer.js

```diff
@@ -40,6 +46,19 @@ export const SensorReducer = (state = initialState(), action) => {
       return { ...state, editingId: id };
     }
 
+    case DOWNLOAD_ACTIONS.SENSOR_DOWNLOAD_SUCCESS: {
+      const { payload } = action;
+      const { sensor } = payload;
+      const { id, currentTemperature } = sensor;
+      const { byId } = state;
+      const oldSensor = byId[id];
+
+      const newSensor = { ...oldSensor, currentTemperature };
+      const newById = { ...byId, [id]: newSensor };
+
+      return { ...state, byId: newById };
+    }
```

Comment:
This is picking up after a download happens to update the `currentTemperature`. When we properly have functionality creating breaches, we can add another case here to pick that up and  display when the last breach was, properly.

Reply:
@katherine-sussol maybe when the breaches are calculated, we could do this too?